source/administration/upgrade.rst

```diff
@@ -24,8 +24,6 @@ Existing install directory - *{install-path}*
   If that command does not produce any results because your version is older, try ``whereis platform`` instead.
 Location of your local storage directory
   The local storage directory contains all the files that users have attached to their messages. If you don't know its location, open the System Console and go to **Files > Storage** and read the value in **Local Storage Directory**. Relative paths are relative to the ``mattermost`` directory. For example, if the local storage directory is ``./data/`` then the absolute path is ``{install-path}/mattermost/data``.
-Owner and group of the install directory - *{owner}* and *{group}*
-  Use the ``ls -l {install-path}/mattermost/bin/mattermost`` command to get the owner and group.

```

Comment:
I'd suggest not removing this line just incase anyone uses a differing setup. 

Maybe add something along the lines of: `If you have followed the server setup from the documentation *{owner}* and *{group}* should both be mattermost.

Reply:
I think this would be too much. I mean user that does it manually should know there owner:group that they give. All others could profit from #2502

Reply:
> I mean user that does it manually should know there owner:group that they give

That's true, but sometimes IT admin roles change and the information gets lost.

My worry is that if the new IT admin then follows the admin doc (with owner=group=mattermost) and it fails, they have no idea why and they panic.

I like the suggestion of having a note after step 9. Admins can skip it if it doesn't apply to them. Maybe we can format it as a note like below

```
.. note:: If you used a different owner and group of the install directory, run ``sudo chown -hR {owner}:{group} tmp/mattermost-upgrade/``

If you're uncertain what owner or group was defined, use the ``ls -l {install-path}/mattermost/bin/mattermost`` command to get the owner and group.
```

Thoughts @svelle or @JtheBAB

Reply:
Good proposal @jasonblais I'd go with this one.